pivy-importer/src/main/groovy/com/linkedin/python/importer/deps/WheelsDownloader.groovy

```diff
@@ -25,18 +27,27 @@ class WheelsDownloader extends DependencyDownloader {
     static final String BINARY_DIST_PACKAGE_TYPE = "bdist_wheel"
     static final String BINARY_DIST_ORG = "wheel"
 
-    WheelsDownloader(String project, File ivyRepoRoot, boolean lenient) {
-        super(project, ivyRepoRoot, lenient)
+    WheelsDownloader(String project, File ivyRepoRoot, DependencySubstitution dependencySubstitution,
```

Comment:
You could also use @InheritConstructors for this class I think.

Reply:
Actually, I decide to pass processedDependencies to this class.